support-frontend/assets/pages/supporter-plus-landing/components/threeTierCard.tsx

```diff
@@ -250,9 +252,24 @@ export function ThreeTierCard({
 							}
 						})}
 					</span>
-					<span css={benefitsPrefixPlus}>plus</span>
 				</div>
 			)}
+			{offers?.description && (
+				<div css={benefitsPrefixCss}>
+					<span>
+						{offers.description.map((stringPart) => {
+							if (typeof stringPart === 'string') {
```

Comment:
non-blocking - it might be worth having the check on strong as the structure of this feels like it's saying "make everything strong, except just strings"

Reply:
Ok, have reversed the check, struggled how to check for an object of exactly `{copy:string, strong:boolean}` so have opted for `typeof stringPart !== 'string'` instead. So the check has been reversed at least. Think done.